src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/resource/LocationResource.kt

```diff
@@ -1138,6 +1138,51 @@ class LocationResource(
     }
   }
 
+  @PutMapping("{id}/used-for-type", produces = [MediaType.APPLICATION_JSON_VALUE])
+  @PreAuthorize("hasRole('ROLE_MAINTAIN_LOCATIONS') and hasAuthority('SCOPE_write')")
+  @Operation(
+    summary = "Update the used for type for a location",
```

Comment:
` summary = "Update the list of `Used For` types at cell location below the specified location",`